### DISCUSSION TITLE
Revision history for project items

### DISCUSSION THREAD

**Discussion by bgmort** [community/community#59767] [Enterprise]
### Select Topic Area

Question

### Feature Area

Projects

### Body

My organization uses GHE for managing our repos and another Agile software product for project management. I am playing around with projects, and hoping to pitch to make the switch to using Github projects for project management. One missing feature that is likely to cause some resistance is the lack of visibility into the history of project items. In our existing software, we can see who made any change and when. As far as I can tell, there's no way to do this either in the UI or with the API. If there is, I'd love to know how, and if not, I'd like to request that it be added in.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mdonges** [comment]
I lately had a project open in two windows.
I updated an Item and saved ti in one window.

Then i updated the same Item from and old version int the second window.


Finally the first changes where completely lost.
I did not find a way to see the items revision history...
I hoped there is some similar feature available like in the wiki, where i can checkout the wiki itself as a git project and look into th internal items revisions. But sadly i had to discover that nothing like this seems to exist and my changes have blown into the ether...without a chance to recover.

**Reply by human-bee** [reply]
I'm also looking for a 'Projects' revision history/log. Were you able to find anything?

**Comment by tianon** [comment]
Yeah, it's kind of wild that we've got edit history for issues, PRs, comments, etc, but not the new projects feature -- it contributes to the feature feeling out-of-place and/or unfinished compared to the rest of the platform (because it doesn't have the same "base level" of functionality).
